app_test.py

```diff
@@ -380,3 +440,42 @@ def testEcho(self):
         ew.on_open()
         ew.on_message('message')
         ew.on_close()
+
+
+class TestOauthBackend(SiteTest):
+    """ Test cases for the oauth.py file"""
```

Comment:
These docstrings aren't consistent with the previous, add a space at the end pls